test/test_components.py

```diff
@@ -12,6 +12,7 @@
 import pandas as pd
 import PIL
 import pytest
+from scipy.io.wavfile import read
```

Comment:
Small stylistic thing, but better to import modules or classes, not methods. (reference: https://google.github.io/styleguide/pyguide.html#22-imports)

Reply:
Good point - done!